researchstudio-sat/webofneeds
# Fix #1682 - retract/reject fails for proposeToCancel

Steps to verify:
`won.protocol.highlevel.GetProposalsTest.oneAgreementProposalToCancelRetractedBeforeAcceptTest() ` uses a conversation with a retract of a proposeToCancel. The expected output therefore is the previously made agreement.

you can edit the input file of that test and remove the retract statement, then the test should fail.

Do the same with the similarly named test for `reject`.

Fixes #1682
